db/db.pl

```diff
@@ -2083,6 +2085,35 @@ sub parseArgs {
     }
     esPost("/_flush/synced", "", 1);
     logmsg "\n";
+    exit 0;
+} elsif ($ARGV[1] eq "rmusers") {
+    showHelp("Invalid number of <hours>") if (!defined $ARGV[2] || $ARGV[2] !~ /^[+-]?\d+$/);
+
+    my $users = esGet("/${PREFIX}users/_search?size=1000");
+    my $rmcount = 0;
+
+    foreach my $hit (@{$users->{hits}->{hits}}) {
+        if (exists $hit->{_source}->{lastUsed} && !$hit->{_source}->{createEnabled} && $hit->{_source}->{enabled}) {
```

Comment:
this works, but you could also just have es do this with something like (not sure totally right), but either way is fine.

/${PREFIX}users/_search?size=1000&q=_exists_:lastUsed+AND+createEnabled:1+AND+enabled:1

https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html

Reply:
👍